docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst

```diff
@@ -0,0 +1,8 @@
+Vivarium Modeling Strategy - Polypill Intervention 
+--------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for polypill intervention
```

Comment:
I expect this to reference the hypertension treatment doc and describe how some ramp positions can be achieved by a polypill; it should say what the effect of polypill is on adherence, and who gets polypill (perhaps with reference to the scenarios.rst description of what the coverage of polypill is in your alternative scenarios.)